### DISCUSSION TITLE
keyboard for mobile

### DISCUSSION THREAD

**Discussion by xavier9203** [responsively-org/responsively-app#544] [General]
Hi there,

First of all, a huge thanks for your work, this is awesome and super useful !!!

My question is the following : on my webapp there are some forms where the user has to write text, and I would like to test the behaviour of my view when the keyboard from the mobile is deployed.
For the moment, when I write something in the form, nothing is changing, it is like on my desktop.

Do you know how to do this ?

Thanks a lot for your help and support !

**Comment by manojVivek** [comment]
@xavier9203 I'm afraid that is not possible right now. This is a long shot, but can you please open an issue for this?
